_projects/nextgen_action_system.md

```diff
@@ -14,10 +14,17 @@ tags:
  - core
 ---
 
-Currently, `Result` objects are passed to action objects as parameters when coala tries to apply an action. This is a bit restrictive as this means that coala needs to have a global set of actions that it tries to apply on all `Result` objects. Sure, coala checks if an action `is_applicable()` before it tries to apply it but it needs to do that for every `Result` that every bear ever generates.
-
-We need a way allow results to define actions by themselves, like `result = Result(actions=[ApplyPatchAction, ...]). Here is the benefit of such an approach:
-* Bears can specify their own actions, for example the GitCommitBear would highly benefit from this (actions could be added for amending commits). Then, the bear writers could get much more creative and generate even more useful bears.
+Currently, `Result` objects are passed to action objects as parameters when
+coala tries to apply an action. This is a bit restrictive as this means that
+coala needs to have a global set of actions that it tries to apply on all
+`Result` objects. Sure, coala checks if an action `is_applicable()` before it
+tries to apply it but it needs to do that for every `Result` that every bear
+ever generates.
+
+We need a way allow results to define actions by themselves, like `result = Result(actions=[ApplyPatchAction, ...]).
+Here is the benefit of such an approach:
+* Bears can specify their own actions, for example the GitCommitBear would highly benefit from this (actions could be
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.